companies/tradedoubler.json

```diff
@@ -0,0 +1,40 @@
+{
+    "slug": "tradedoubler",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "ads"
+    ],
+    "name": "Tradedoubler AB",
+    "runs": [
+        "Tradedoubler GmbH",
+        "Adnologies",
+        "Interactive Marketing Works Ltd"
+    ],
+    "address": "Birger Jarlsgatan 57A\n113 56 - Stockholm\nSweden\n",
+    "phone": "+46 8 405 08 00",
+    "email": "privacy@tradedoubler.com",
+    "sources": [
+        "http://performancein.com/news/2015/01/26/tradedoubler-completes-deal-adnologies/",
+        "https://en.wikipedia.org/wiki/Tradedoubler",
+        "https://www.tradedoubler.com/en/privacy-policy/",
+        "https://www.tradedoubler.com/en/about-us/"
+    ],
+    "required-elements": [
+        {
+            "desc": "TradedoublerGUID",
```

Comment:
Do we really want to start doing that? I feel like most users will be overwhelmed by that. I would have just made this a tracking company and let them deal with explaining how to access the cookies.

Reply:
They explicitly tell you this as identifying information in their privacy policy. That's why I included it.

Reply:
Most of the tracking companies do that. The problem I see is that we are not even remotely equipped to explain to users how to get the cookie values.

Even if they see the comment (which is already unlikely), it is in English and most users will **definitely** not know what "You can find the GUID in the respective cookie." is supposed to tell them.